quay-enterprise/insert-custom-cert.md

```diff
@@ -87,4 +87,25 @@ Add an entry for the cert and paste the full base64 encoded string under the ent
 c1psWGpqeGlPQmNEWkJPMjJ5d0pDemVnR2QNCnRsbW9JdEF4YnFSdVd3PT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
 ```
 
+Add the env variable `REQUESTS_CA_BUNDLE` to the quay deployment: 
+
+```
+kubectl --namespace quay-enterprise edit deploy/quay-enterprise-app 
+
+```
+
+The `REQUESTS_CA_BUNDLE` env should be defined under the `containers:` section: 
+
+```
+      containers:
+      - name: quay-enterprise-app
+        image: quay.io/coreos/quay:v2.6.1
+        ports:
+        - containerPort: 80
+       env: 
+       - name: REQUESTS_CA_BUNDLE 
+	 value: /conf/stack/custom-cert.crt
```

Comment:
Indentation is off

Reply:
I think back it all the way to the left, please. we don't typically indent snippets like this - just start fully outdented, and go from there. like so:
```
containers:
- name: quay-enterprise-app
  image: quay.io/coreos/quay:v2.6.1
  ports:
  - containerPort: 80
 env: 
 - name: REQUESTS_CA_BUNDLE 
value: /conf/stack/custom-cert.crt
```